rmw_test_fixture_implementation/rmw_test_fixture_implementation/__init__.py

```diff
@@ -14,10 +14,16 @@
 
 from contextlib import AbstractContextManager
 
-from rmw_test_fixture_implementation._rmw_test_fixture_implementation \
-    import rmw_test_isolation_start
-from rmw_test_fixture_implementation._rmw_test_fixture_implementation \
-    import rmw_test_isolation_stop
+from rpyutils import add_dll_directories_from_env
+
+# Since Python 3.8, on Windows we should ensure DLL directories are explicitly
+# added to the search path.
+# See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
+with add_dll_directories_from_env('PATH'):
```

Comment:
Does this need to be gated on executing on Windows, or does this effectively become a no-op on non-Windows platforms?

Reply:
The gating happens in the function: https://github.com/ros2/rpyutils/blob/9587b7668356937182b600918ecc7c13633fa1f6/rpyutils/add_dll_directories.py#L50-L51

Reply:
Ah, should have looked at that, thanks!